src/service.py

```diff
@@ -176,6 +176,12 @@ def stop(self):
                 print("Killing orderly container failed - continuing")
                 pass
         compose.stop(self.settings)
+        print("Wiping static file volume")
```

Comment:
Occurred to me that we also need to delete files from the volume that are not in the config, hence wiping this volume each time. Kind of means that a volume is the wrong technology for this, if we don't want it to persist, but copying files into the container would be a faff.

Reply:
The alternative way would be a new volume utility for clearing the volume just before doing a copy (would be the same lightweight container approach running `rm -rf /volume/*`) but this way seems no less complicated